src/sage/interfaces/quit.py

```diff
@@ -37,12 +37,8 @@ def sage_spawned_process_file() -> str:
         sage: len(sage_spawned_process_file()) > 1
         True
     """
-    # This is the old value of SAGE_TMP. Until sage-cleaner is
-    # completely removed, we need to leave these spawned_processes
-    # files where sage-cleaner will look for them.
-    d = os.path.join(DOT_SAGE, "temp", HOSTNAME, str(os.getpid()))
-    os.makedirs(d, exist_ok=True)
-    return os.path.join(d, "spawned_processes")
+    from sage.misc.temporary_file import tmp_dir
```

Comment:
Is it much work to migrate this to the built-in python tempdir?

(Triggered by the fact the last PR that touched the cleaner interface https://github.com/sagemath/sage/issues/33213 notes "Afterward, the custom functions tmp_dir() and tmp_filename() can be deprecated in favor of tempfile.TemporaryDirectory() and tempfile.NamedTemporaryFile().")

Reply:
Okay, thanks for checking!